server/app.js

```diff
@@ -0,0 +1,29 @@
+const { join } = require('path');
+const express = require('express');
+const cookieParser = require('cookie-parser');
+
+const compression = require('compression');
+const morgan = require('morgan');
+
+const controllers = require('./controllers');
+
+const app = express();
+app.use(express.json());
+app.use(cookieParser());
+app.set('port', process.env.PORT || 5000);
+
+if (process.env.NODE_ENV === 'development') {
+  app.use(morgan('dev'));
+}
+
+app.use(express.json());
+app.use(compression());
```

Comment:
No, we should not 
because URL urlencoded used for `application/x-www-form-urlencoded` data type 
but in our case, we use Axios to sed request, not the actual form